src/components/responsive/DesktopPopup.js

```diff
@@ -138,7 +138,7 @@ const DesktopPopup = ({
                 <List className='submenu'>
                     <List.Item>
                         <List.Icon as={Image} src={AccountImage} />
-                        <List.Content as={Link} to='/profile' onClick={handleClose}>
+                        <List.Content as={Link} to={`/${account.accountId}`} onClick={handleClose}>
```

Comment:
URL should have some prefix before `${account.accountId}` or we'll end up having name clashes for accounts and routes in wallet.

Reply:
The route is listed last in `Routing` for just this reason

Reply:
That still doesn't allow to have e.g. `send-money` as account name.

Reply:
Ah, great point – maybe just an `@` before the username would be a good approach? Should we get design insight on this?

Reply:
@chadoh let's just make it `/account/${account.accountId}`